### ISSUE TITLE
Dbeaver connection info isnt complete.

### ISSUE THREAD

**Issue by chrispward** [Teradata/quickstarts#163]
1. Doc shows info for using mac.  Windows info is different (more people use windows than macs)
2. No info on how to use other logmech's like LDAP.

**Comment by Daniel-Itzul** [comment]
Will take a look.

**Comment by Daniel-Itzul** [comment]
@chrispward 
Thank you very much for your comments. 
1. We didn't identify a material difference between the setting of the connection between mac and windows. Specifically on the required steps, defining url, user and password. 
Wonder if you've found a material step there that differed and made the content confusing, appreciate your elaboration on that. 
2. Noted, we will expand the quickstart on this. Let you know when this is added and we'll close the issue then.

**Comment by chrispward** [comment]
specifically the thing that is missing is details on setting the logmech.
ie TD2 is the default. LDAP is probably 2nry.  Detailing how to set this who be great.
FYI. I figured it out by going to the td jdbc and looking at the jdbc details.. basically add /LOGMECH=LDAP to the URL string is how i did it. Not sure if there is a setting...  The point is I had to go to another site to figure it out. I would image its preety common.  
https://teradata-docs.s3.amazonaws.com/doc/connectivity/jdbc/reference/current/frameset.html
